extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheHibernateResourceProcessor.java

```diff
@@ -106,27 +127,13 @@ void build(CombinedIndexBuildItem index,
 
         PanacheJpaEntityEnhancer modelEnhancer = new PanacheJpaEntityEnhancer(index.getIndex(), methodCustomizers);
         Set<String> modelClasses = new HashSet<>();
-        // Note that we do this in two passes because for some reason Jandex does not give us subtypes
-        // of PanacheEntity if we ask for subtypes of PanacheEntityBase
-        // NOTE: we don't skip abstract/generic entities because they still need accessors
-        for (ClassInfo classInfo : index.getIndex().getAllKnownSubclasses(DOTNAME_PANACHE_ENTITY_BASE)) {
```

Comment:
Looks like you ignored the first note, so I assume this limitation is no longer necessary?
I think you should have kept the second note and FIXME comment.

Reply:
The tests passed locally and TBH I've never observed such a limitation in jandex.

The FIXME was not present in the mongo version so I though it's no longer a problem. 

I can keep the second note and FIXME of couse ;-).

Reply:
Yes please. Glad to know the tests pass with this, then. I mean, that code was there initially for a reason, so something must have happened, but glad we can get rid of it.

Reply:
Hopefully the test coverage is good enough ;-)

Reply:
This would have happened if the jar containing PanacheEntityBase was not indexed. I think originally this was the case which is why this would not have worked, but it is indexed now.